src/mrinufft/io/utils.py

```diff
@@ -60,3 +60,28 @@ def siemens_quat_to_rot_mat(quat):
         R[2] = -R[2]
     R[-1, -1] = 1
     return R
+
+
+def remove_extra_kspace_samples(kspace_data, num_samples_per_shot):
+    """Remove extra samples from k-space data.
+
+    This function is useful when the k-space data has extra samples
+    mainly as ADC samples at only upto
+
```

Comment:
finish the sentence ?

Reply:
Lol this is what happens when someone calls me to show their fancy results while writing code :P